app/blueprints/restaurant_blueprint.rb

```diff
@@ -0,0 +1,3 @@
+class RestaurantBlueprint < Blueprinter::Base
```

Comment:
This class is used by `RestaurantsController` to return only restaurant names (listed as a product requirement) via the endpoint